### PR TITLE
feat: extended tracking to be able to see recurring users

### PR DESCRIPTION
**Pull request by SimonTheSourcerer**
### Description
Extended analytics to allow fixed session ids, which allow us to differentiate between recurring and one-time users.
Also extended the analytics popup to properly explain this.

### CODE REVIEW

File: Editor/Analytics/AnalyticsSetup.cs [hunk 1]
```diff
@@ -17,38 +17,60 @@ internal class AnalyticsSetup
 
         static AnalyticsSetup()
         {
+            //SetupTrackingPopup.Open();
```
Comment (Gusinuhe) [inline Editor/Analytics/AnalyticsSetup.cs:20]: Is this a comment? 🤔

File: Editor/Analytics/GoogleTracker.cs [hunk 2]
```diff
@@ -19,7 +17,8 @@ public override void Send(AnalyticsEvent data)
 
         private void PostData(string uri)
         {
-            webClient.GetAsync(uri);
+            //webClient.GetAsync(uri);
+            Debug.LogWarning(uri);
```
Comment (Gusinuhe) [inline Editor/Analytics/GoogleTracker.cs:21]: 🤔

File: Editor/Analytics/AnalyticsSetup.cs (hunk 1 above)
Comment (SimonTheSourcerer) [reply Editor/Analytics/AnalyticsSetup.cs:20]: no, should be removed :D

File: Editor/Analytics/AnalyticsSetup.cs [hunk 3]
```diff
@@ -22,33 +22,52 @@ static AnalyticsSetup()
             {
                 return;
             }
-
-            List<string> args = Environment.GetCommandLineArgs().ToList();
-            if (args.Contains("-no-tracking"))
+            // Can be used by ci to deactivate tracking.
+            if (Environment.GetCommandLineArgs().Contains("-no-tracking"))
```
Comment (Gusinuhe) [inline Editor/Analytics/AnalyticsSetup.cs:26]: 👌

**Pull request merged**